app/assets/stylesheets/components/_c-analysis-header.scss

```diff
@@ -0,0 +1,75 @@
+// ANALYSIS HEADER
+// <header class="c-analysis-header">
+//   <div>
+//     <ul class="list">
+//       <li class="list-item">
+//         <span class="title">Country</span>
+//         <span class="value"><%= @analysis.geo_location.country %></span>
+//       </li>
+//       <li class="list-item">
+//         <span class="title">Region</span>
+//         <span class="value"><%= @analysis.geo_location.state %></span>
+//       </li>
+//     </ul>
+//     <ul class="list">
+//       <li class="list-item">
+//         <span class="title">Crop</span>
+//         <span class="value"><%= @analysis.crop.capitalize %></span>
+//       </li>
+//       <li class="list-item">
+//         <span class="title">Area</span>
+//         <span class="value"><%= @analysis.area %> <em>ha</em></span>
+//       </li>
+//       <li class="list-item">
+//         <span class="title">Annual Production</span>
+//         <span class="value"><%= @analysis.yield %> <em>kg. ha<sup>-1</sup> yr<sup>-1</sup></em></span>
+//       </li>
+//     </ul>
+//   </div>
+// </header>
+
+
+.c-analysis-header {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  padding: rem(20px) 0;
+  border-bottom: 1px solid rgba($font-color-2, 0.1);
+
+  .list {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin: 0 0 rem(15px);
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  .list-item {
+    margin: 0 rem(20px) 0 0;
+
+    &:last-child {
+      margin: 0;
+    }
+
+    .title {
```

Comment:
Oh. 

We already have those text styles in the `text.sccs` file.
In order to keep all the different text styles in the same place, for future reference, I think you should use them.